flink/v1.20/flink/src/main/java/org/apache/iceberg/flink/sink/FlinkManifestUtil.java

```diff
@@ -66,6 +67,18 @@ static List<DataFile> readDataFiles(
     }
   }
 
+  static List<DeleteFile> readDeleteFiles(
+      ManifestFile manifestFile, FileIO io, Map<Integer, PartitionSpec> specsById)
+      throws IOException {
+    List<DeleteFile> deleteFiles = Lists.newArrayList();
+    try (CloseableIterable<DeleteFile> deleteFileIterable =
+        ManifestFiles.readDeleteManifest(manifestFile, io, specsById)) {
+      Iterables.addAll(deleteFiles, deleteFileIterable);
+    }
+
+    return deleteFiles;
+  }
+
```

Comment:
We can revert these changes too?
I think we don't need to change this file after the simplification